.contributing/the-process.md

```diff
@@ -61,27 +61,24 @@ For each use-case there will be three types of digital goodness:
 
     -   Sequence Diagram
 
-Create: 
+Inside Google Drive Folder "x. *Section*: Sections Planning (CL)", create:
 
--   An Asana task for each use case, linked to the GitHub issue and Google doc (see below) 
+-   A Models file
 
--   A Drive folder for each use case, called "x. *Use Case* < *Section* (CL)" and in that folder, a Google Doc for each use case: 
+      -   Create a UML  - ([example](https://docs.google.com/drawings/d/1-X-aZdVrsuFItxlwkJqusOINdk0kYb9dl3fgLTQVz94/template/preview?usp=drive_web))
+      
+      -   Add title - "Models < x. *Use Case* < *Section* (CL)"
 
-    -   Models 
-
-        -   "x. *Use Case* < Models < *Section* (CL)"
-        
-        -   Google Drawing 
+-   A Drive folder for each use case, called "x. *Use Case* < *Section* (CL)" and in that folder, create:
 
-        -   UML Diagram ([example](https://docs.google.com/drawings/d/1-X-aZdVrsuFItxlwkJqusOINdk0kYb9dl3fgLTQVz94/template/preview?usp=drive_web)) 
+    -   A User Flow:
 
-    -   A User Flow 
+        1. Create a folder to hold all files needed to create a user flow diagram
+            - After creating the user flow diagram, all files in this folder dont need to be looked at. So keeping them seperate will reduce clutter and make it easier to look for the files we need to look at when creating github issues and coding! 
+            - Add title: "User Flow Files < x. *Use Case*  < *Section* (CL)"
 
-        1. Create a folder to hold all files needed to create a user flow
-            - Add title: "x. *Use Case* < User Flow files < *Section* (CL)"
```

Comment:
changed title from "x. *Use Case* < User Flow files < *Section* (CL)" to "User Flow Files < x. *Use Case*  < *Section* (CL)"

moving "User flow files" to the front of the title made it easier to read when in google drive as the title is so long it gets cut off.

here is before:
![drive_before](https://user-images.githubusercontent.com/49229277/102043497-aacb5b80-3e28-11eb-92f1-4051dc4a0047.PNG)

and after:
![drive_pic_after](https://user-images.githubusercontent.com/49229277/102043444-840d2500-3e28-11eb-8ba8-4436ef83b3fa.PNG)

Reply:
sweet!!!